modules/neo4j/examples_test.go

```diff
@@ -15,7 +15,7 @@ func ExampleRunContainer() {
 	testPassword := "letmein!"
 
 	neo4jContainer, err := neo4j.RunContainer(ctx,
-		testcontainers.WithImage("docker.io/neo4j:15.2-alpine"),
+		testcontainers.WithImage("docker.io/neo4j:4.4"),
```

Comment:
Good catch! And it probably did not fail because of https://github.com/testcontainers/testcontainers-go/pull/1903/files#diff-cdacf25279bf22e2de635c654d0bb2c37c1b5699d8c13c07da3c0886b2087218L94 🤦